modules/campaignion_donation_type/campaignion_donation_type.install

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Update hooks for the campaignion_donation_type module.
+ */
+
+/**
+ * Enable campaignion_donation_amount.
+ */
+function campaignion_donation_type_update_1() {
+  if (!module_enable(['campaignion_donation_amount'], TRUE)) {
+    new DrupalUpdateException("Unable to enable ‘campaignion_donation_amount’. Please verify it (and all it's dependencies) can be found by Drupal.");
+  }
```

Comment:
Are you not setting the `campaignion_donation_amount_currency_symbol` variable to "€" for a specific reason?

Reply:
Because it's the default value, makes sense!